apps/organizations/utils.py

```diff
@@ -29,3 +35,45 @@ def get_user_organizations_with_attribute(user, key):
         organization__attributes__value=None
     )
     return organizations
+
+
+def invite_person_to_organization(request, invitation):
+    """Invite person to organization
+
+    Sends invitation e-mail to given person.
+    If prelaunch app is installed, in prelaunch mode and auto adding to waiting list enabled,
+    it adds early access code to invitation link.
+    This function can also be used for re-send invitation links.
+
+    Args:
+        request (Request): Request object. It is needed to build full URL and get current site
+        invitation (BaseAbstractOrganizationInvitation): Invitation object can be come from form.
+    """
+    early_access_code = None
+    if (
+        apps.is_installed('htk.apps.prelaunch')
+        and is_prelaunch_mode()
+        and htk_setting('HTK_ORGANIZATION_INVITATION_AUTO_ADD_TO_WAIT_LIST')
```

Comment:
Is this setting necessary? If it were `False`, how would this even work?

Reply:
If it is `False` it will not append any `early_access_code` to invitation. I might help to prevent the people in the Team which is not an engineer and accidentally inviting someone before it is ready while testing. Maybe it is a long shot?

Reply:
Yeah, IMO, not necessary, just remove

Reply:
Done.